examples/language-features/imports.qnt

```diff
@@ -23,21 +19,23 @@ module B {
 
   // Import introduces names in the module lookup table,
   // but it is not transitive. Hence, we cannot access sqr here.
-  // val test_sqr = sqr(2) == 4
-  //                ^^^ error: sqr is not available
+  // val test_test_sqr = sqr(2) == 4
+  //                     ^^^^^^ error: Failed to resolve name sqr in definition for test_test_sqr, in module B
 
   // however, we can access test_sqr
   val test_test_sqr = test_sqr
+
+  // To enable transitivity, we need to export the names that we want to make
+  // available when importing the current module
+  export A.*
```

Comment:
That's interesting. I would expect `export *` here, don't know whether it is even supported by our syntax. Is not it a bit strange that the namespace `A` is available within `A` itself? On the other hand, I think this is how PLs are doing that, so maybe it's ok.

Reply:
This is inside module `B`, not module `A`. `export *` is the default implicit behavior, and is not supported by our syntax. You can only export other modules or names from other modules.